BoxContentSDK/BoxContentSDK/Operations/BOXAPIDataOperation.m

```diff
@@ -294,6 +294,17 @@ - (void)progressWithTotalBytesWritten:(int64_t)totalBytesWritten totalBytesExpec
     }
 }
 
+- (void)didFinishDownloadingToURL:(NSURL *)location
+{
+    if (self.destinationPath != nil) {
+        NSError *error = nil;
+        [[NSFileManager defaultManager] moveItemAtPath:location.path toPath:self.destinationPath error:&error];
+        if (self.error == nil) {
```

Comment:
for this type of operation, we don't expect it to be. It's more of a safe check to honor existing error if it comes before this.